src/main/java/pl/zimi/zadanka/Zad6.java

```diff
@@ -2,30 +2,41 @@
 
 public class Zad6 {
     public int removeDuplicates(int[] nums) {
-        int Len = 0;
+        if (nums.length == 0) return 0;
+        if (nums.length == 1) return 1;
+
         int numsLen = nums.length - 1;
         int numsMin = nums[0];
         int numsMax = nums[numsLen];
-        int arrayRank = numsMax - numsMin + 1;
-//        System.out.println("arrayRank = " + arrayRank);
+        int uniqueNumbersCounter = 0;
+        int tempPreviousNumber = -1;
+        int i = 0;
+        for (int number : nums) {
```

Comment:
do tej pętli wystarczy jedna linia żeby rozwiązać to zadanie

Reply:
wiem, że można było to wykonać inaczej. Zmieniać?

Reply:
jeżeli szybko umiesz to zmienić to tak
jeżeli masz myśleć długo to spróbuję Ciebie naprowadzić na rozwiązanie przy kolejnej rozmowie...
w podobny sposób naprowadzają czasem na rozmowach... rekruterzy są świadomi że nie zawsze wpadniemy na coś z marszu... czasem potrzeba chwila czasu którego nie ma na rozmowie...dlatego podrzucają jakieś sugestie...
jeśli nie masz pomysłu i myśląc mniej niż 15 min rozwiązanie nie przychodzi Ci do głowy... pogadamy o tym przy kolejnej lekcji